src/sentry/static/sentry/app/components/charts/components/legend.jsx

```diff
@@ -1,25 +1,17 @@
 import 'echarts/lib/component/legend';
 import 'echarts/lib/component/legendScroll';
-
-const DEFAULT_TRUNCATE_LENGTH = 80;
+import {truncationFormatter} from '../utils';
 
 export default function Legend({truncate, ...props} = {}) {
-  let truncateLength =
-    truncate && typeof truncate === 'number' ? truncate : DEFAULT_TRUNCATE_LENGTH;
+  const formatter = value => {
+    return truncate ? truncationFormatter(value, truncate) : value;
```

Comment:
You can simplify this to remove the ternary since `truncationFormatter` handles the case when `truncate` is falsey.